### ISSUE TITLE
Code and Doc ownership in Github and elsewhere matches documented governance roles

### ISSUE THREAD

**Issue by linsun** [in-toto/community#40]
Another required item for graduated project:

[ ] Code and Doc ownership in Github and elsewhere matches documented governance roles

I need help to check on this since I am not repo maintainer/admin, would be good to provide some screenshots or data if you can.

cc @SantiagoTorres

**Comment by trishankatdatadog** [comment]
@linsun Do you need to see, for example, the screenshots for which individuals and teams have what types of access to which repos?

**Comment by linsun** [comment]
Yep, I'd like to see it documented somewhere, which can be in this GitHub issue or sent to me privately via CNCF slack.

**Comment by JustinCappos** [comment]
I spoke with @linsun and walked her through the permissions over a slack call.   We've addressed this.
